translations/base-de.yaml

```diff
@@ -1128,10 +1128,10 @@ keybindings:
         rotateToDown: "Rotieren: Nach unten zeigend"
         rotateToRight: "Rotieren: Nach rechts zeigend"
         rotateToLeft: "Rotieren: Nach links zeigend"
-        constant_producer: Constant Producer
-        goal_acceptor: Goal Acceptor
-        block: Block
-        massSelectClear: Clear belts
+        constant_producer: Item-Produzent
+        goal_acceptor: Ziel Akzeptor
```

Comment:
Da musst du noch etwas besseres finden! x)

Reply:
Übersetzung aus der orginal Übersetzung vom DLC übernommen. Gleichgelassen, um es überall gleich zu bezeichnen.

Reply:
Aber `Ziel Akzeptor` ist weder orthographisch richtig, noch eine schöne freie Übersetzung

Reply:
Entweder du schreibst es mit Bindestrich, oder du setzt die Worte zusammen, wie man das so in der deutschen Sprache macht. Das habe ich übrigens auch bei der restlichen Übersetzung so versucht. Also so wenige Bindesriche wie möglich und nur zwischen deutzschen und rein englischen, nicht eingedeutschen Begriffen.